cmd/sync_test.go

```diff
@@ -0,0 +1,134 @@
+package cmd
+
```

Comment:
Guard this file with an integration tag.
make tests shouldn't run tests that have dependencies.

Reply:
This test also must move under "tests/e2e" or something like that.